### PR TITLE
 Alex Yang CV Bootcamp

### PR DESCRIPTION
**Pull request by YMRTZ**

### CODE REVIEW

**Review by mikashaw — CHANGES_REQUESTED** [review]
Hey Alex! Would you mind just adding loss and accuracy plot images and the I can review, thanks :)

**Comment by YMRTZ** [comment]
Graphs

**Review by mikashaw — CHANGES_REQUESTED** [review]
Hey Alex, great job on the bootcamp! For this team, we require a 65- 70% min accuracy to have it be considered complete - I would recommend adding some convolutions and max pooling to your neural network! (Try a CNN) - if you need any help, let myself or aydan know!

**Comment by YMRTZ** [comment]
Using a CNN doesn't seem to have done much for the accuracy. Any tips on how to improve the convolution bit?

File: main.py [hunk 1]
```diff
@@ -55,16 +55,40 @@ def unpickle(file):
...
+      # nn.Linear(32*32,512),
+      # nn.ReLU(),
+      # nn.Linear(512,512),
+      # nn.ReLU(),
+      # nn.Linear(512,10),
   def forward(self,x):
-    logits = self.linearReLUStack(x)
+    logits = torch.reshape(x, (32,32,3))
+    logits = logits.permute(2,0,1)
+    logits = self.conv1(logits)
```
Comment (mikashaw) [inline main.py:85]: It looks like your using convolutions without pooling in between, as well as not adding enough convolutiopns for the image features to be extracted. Normally, A typical CNN looks like: Cnv2D -> MaxPool -> Conv2D -> Max Pool -> Flatten -> Dense layers with activation -> final softmax or sigmoid layer. I would try (1) adding max pooling  (I see it in the class initialization but not the forward() method), and (2) tweaking the CNN parameters as having only 1 dim in the argument  (e.g for your second convolution ) will not do much! Here is a pretty helpful article going over CNN architecture https://medium.com/thecyphy/train-cnn-model-with-pytorch-21dafb918f48 with sample code, you don't need to follow this - it just has some helpful tips about how a typical CNN should look like et cetera!

**Review by mikashaw — CHANGES_REQUESTED** [review]
Hey Alex! I've written a comment for this pull request in the main.py file, hope that helps!
